### PR TITLE
Add tkn approvaltask 😻 

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
This patch bootstraps CLI and adds implementation of List Command

* List all the approval task in the default namespace
  * If flag provided the list the approval task in that particular namespace
* Flags 
  * -A, –all-namespace → To list approvaltask from all namespaces
* Global Flags
  * –namespace  → namespace to use
* UX 

```bash
tkn approvaltask list
tkn at ls
```

Output

```bash
NAME   NumberOfApprovalsRequired   PendingApprovals   Rejected   STATUS
cr-1   2                           2                  0          Pending
cr-2   2                           0                  1          Rejected
cr-3   2                           0                  0          Approved
```

### CODE REVIEW

File: pkg/cli/cmd/list/list.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	taskGroupResource = schema.GroupVersionResource{Group: "openshift-pipelines.org", Resource: "approvaltasks"}
+)
+
+var ConditionColor = map[string]color.Attribute{
+	"Rejected": color.FgHiRed,
+	"Approved": color.FgHiGreen,
+	"Pending":  color.FgHiBlue,
+}
+
+const listTemplate = `{{- $at := len .ApprovalTasks.Items }}{{ if eq $at 0 -}}
```
Comment (piyush-garg) [inline pkg/cli/cmd/list/list.go:33]: This table makes not much sense for me. Too much information repeated. We can go for something like

```
NAME	   Approvals 	       STATUS
foo 	           2/3            	       Wait
bar 	           4/4            	       Approved
```

**Review by piyush-garg — CHANGES_REQUESTED** [review]

File: pkg/cli/cmd/list/list.go (hunk 1 above)
Comment (piyush-garg) [reply pkg/cli/cmd/list/list.go:33]: As we are supporting only single reject, showing a column does not make any sense

File: pkg/cli/cmd/list/list.go [hunk 2]
```diff
@@ -1,17 +1,117 @@
...
 
-func Root() *cobra.Command {
+type ListOptions struct {
+	AllNamespaces bool
+}
+
+var ConditionColor = map[string]color.Attribute{
+	"Rejected": color.FgHiRed,
+	"Approved": color.FgHiGreen,
+	"Pending":  color.FgHiBlue,
```
Comment (piyush-garg) [inline pkg/cli/cmd/list/list.go:30]: this should be yellow/orange :D

File: pkg/cli/cmd/list/list.go (hunk 1 above)
Comment (koustavsaha) [reply pkg/cli/cmd/list/list.go:33]: @piyush-garg I am fine with it as long as its clear that in the example of 0/3 and status as "rejected" someone from the approval list has rejected it.

**Pull request merged**